workflows/snakemake/README.md

```diff
@@ -0,0 +1,66 @@
+# Snakemake
+
+```
+# Install micromamba on snellius
+"${SHELL}" <(curl -L micro.mamba.pm/install.sh)
+# During install, set conda-forge as default channel
+source ~/.bashrc
+
+# Create environment (pin 3.11 for ecmwflibs dependency of w2w)
+micromamba create --name snakemake python=3.11 "numpy<2" bioconda::snakemake -y
+micromamba activate snakemake
+pip install git+https://github.com/matthiasdemuzere/w2w@add_wrf_version
+
+# Make sure netcdf is available for WPS and WRF (these instructions are specific to Snellius)
+module load 2023
+module load netCDF-Fortran/4.6.1-gompi-2023a  # also loads gcc and gompi
+export NETCDF=$(nf-config --prefix)
+
+# Run snakefile in this dir
```

Comment:
Might be good to mention the need to change the workdir at the top of the snakefile before running.

Reply:
Outdated